polkadot/runtime/polkadot/src/lib.rs

```diff
@@ -1505,6 +1506,18 @@ pub mod migrations {
 		type PalletName = TipsPalletName;
 	}
 
+	pub struct ParachainsToUnlock;
+	impl Contains<ParaId> for ParachainsToUnlock {
+		fn contains(id: &ParaId) -> bool {
+			let id: u32 = (*id).into();
+			// polkadot parachains/parathreads that are locked and never produced block
+			match id {
+				2003 | 2015 | 2036 | 2055 | 2097 | 2106 | 3336 | 3338 => true,
+				_ => false,
```

Comment:
Same

Reply:
@SBalaguer

Reply:
With the condition of Parachains/Parathreads being locked and never producing one single block, I have 10 more paraIDs for Polkadot:

`2003 | 2015 | 2017 | 2018 | 2025 | 2027 | 2028 | 2036 | 2038 | 2053 | 2055 | 2090 | 2097 | 2106 | 3336 | 3338 | 3340 | 3342`

The ones I'm adding are: 2,017, 2,018, 2,025, 2,027, 2,028, 2,038, 2,053, 2,090, 3,342. My only comment/question is what's the final definition for Crowdloans, as 2,025 has an ongoing Crowdloan and is currently bidding for a slot.

Reply:
Confirmed @SBalaguer's list is correct.

I think we should unlock 2025. The least bidding is for 14, which is 53 days away. So if we think this upgrade will enact before that, than it is fine. Otherwise we should avoid unlock it.